tools/go-cli/go-whisk/whisk/shared.go

```diff
@@ -38,4 +38,5 @@ type Parameters *json.RawMessage
 type Limits struct {
     Timeout int `json:"timeout,omitempty"`
     Memory  int `json:"memory,omitempty"`
+    Logsize int `json:"logs"`
```

Comment:
@ChrBi and me played a bit with it. They are not sent at all (as they weren't with `0` set). Note that the server will decline `0` for memory and timeout limits.